src/layers/core/line-layer/line-layer.js

```diff
@@ -49,15 +49,13 @@ export default class LineLayer extends Layer {
   }
 
   draw({uniforms}) {
-    const {gl} = this.context;
-    const lineWidth = this.screenToDevicePixels(this.props.strokeWidth);
-    gl.lineWidth(lineWidth);
-    this.state.model.render(uniforms);
-    // Setting line width back to 1 is here to workaround a Google Chrome bug
-    // gl.clear() and gl.isEnabled() will return GL_INVALID_VALUE even with
-    // correct parameter
-    // This is not happening on Safari and Firefox
-    gl.lineWidth(1.0);
+    const {strokeWidth} = this.props;
+    const {viewport: {width, height}} = this.context;
+
+    this.state.model.render(Object.assign({}, uniforms, {
+      screenSize: [width, height],
```

Comment:
I would love it if the screen dimensions are provided as default uniforms. `IconLayer` and `ScreenGridLayer` use them as well.